docs/07-templates/01-overview.mdx

```diff
@@ -7,6 +7,18 @@ hide_table_of_contents: true
 draft: false
 ---
 
-# Grafana Template Components
+# Grafana Design System Templates
```

Comment:
Small nit — if we're talking about design system offerings we should refer to it by name: Saga Design System Templates